src/classes/STG_PanelDataImportAdvancedMapping_CTRL.cls

```diff
@@ -47,6 +47,12 @@ public with sharing class STG_PanelDataImportAdvancedMapping_CTRL extends STG_Pa
     private static final String DEFAULT_FIELD_MAPPING_SET_RECORD_NAME =
```

Comment:
These static final vars that hold details about mapping sets, etc., should probably be stored elsewhere, not in the controller for this page.  Maybe in the BDI_FieldMapping class?